CONTRIBUTING.md

```diff
@@ -41,17 +41,18 @@ merged - if you have ten small, unrelated changes, then go ahead and submit ten
 
 # How to release
 
-- Be sure to be rebased on `master` branch
-- Edit `osc_sdk/sdk.py` and update `SDK_VERSION` following [semantic versioning 2.0.0](https://semver.org/)
+- Be sure to fetch upstream master `git fetch origin`
```

Comment:
`git fetch origin; git rebase origin/master`

Reply:
No need to rebase as next line creates a new fresh branch based on origin/master. This avoid possible local conflict and commits which are not been pushed outside the release procedure.